src/tomczyk-kamil/Fsd.Kamil.Cs/Fsd.Kamil.Cs.Ex3/Fsd.Kamil.Cs.Ex3/Fsd.Kamil.Cs.Ex3/ProductMagazine/Menu.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Fsd.Kamil.Cs.Ex3.ProductMagazine
+{
+    public class Menu : Text
+    {
+        public char LeaveOrNot;
```

Comment:
jeżeli to propercja to dodaj {get; set;}, jeżeli to pole to ustaw private i zmień format nazwy na "_zmienna"